sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/InnerError.java

```diff
@@ -43,27 +40,21 @@
     /*
      * Inner error contains more specific information.
      */
-    @JsonProperty(value = "innerError")
-    private InnerError innerError;
+    @JsonProperty(value = "innererror")
+    private InnerError innererror;
```

Comment:
nitpick: Since you're changing the getters and setters of InnerError to have the casing Innererror, it might be worth it to change this property

Reply:
offline discussed. This is swagger changes.